### DISCUSSION TITLE
weather bug thing where sky goes black

### DISCUSSION THREAD

**Discussion by ghostinhost** [Monika-After-Story/MonikaModDev#7770] [QA - tech support + general] [answered]
I have looked at the responses to the few people who have asked about this and tried not using windowed mode but the issue still persists it used to occur randomly but now it is constant and none of the normal tricks will make it go away is there anything I can do that might resolve this issue

**Comment by Booplicate** [comment, marked as answer]
This is an issue with RenPy, we can't fix it. We only can advice to not minimize the game since that's known to cause this bug as well.

**Comment by ghostinhost** [comment]
thanks for letting me know
